packages/indicators/src/__tests__/__integration/fixtures/arithmeticAnalyticsFixtures.ts

```diff
@@ -32,7 +32,7 @@ const ARRAY_SURVEY_RESPONSES: ArraySurveyResponse[] = [
   ['Births', 'GR', '2019-01-01', { Female: '1', Male: '2' }],
   ['Births', 'GR', '2020-11-30', { Female: '3', Male: '4' }],
   ['Births', 'GR', '2020-12-29', { Female: '60', Male: '80' }],
-  ['Births', 'GR', '2020-12-30', { Female: '6', Male: '8' }],
+  ['Births', 'GR', '2020-12-31', { Female: '6', Male: '8' }],
```

Comment:
The existing value (`30`) would also work... `30` was used to enable a match with the previous month (`2020-11-30`), but after https://github.com/beyondessential/tupaia/pull/1941 any month day can be used (which is the desired behaviour)

Reply:
A nice improvement :-)